documentation/src/main/asciidoc/topics/con_protobuf_encoding.adoc

```diff
@@ -0,0 +1,52 @@
+[id='protobuf-encoding_{context}']
+= Protobuf cache encoding
+Protocol Buffers (Protobuf) is a lightweight binary media type for structured data.
+As a cache encoding, Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages and remote server endpoints.
```

Comment:
I'm not sure the meaning of "remote server endpoints" would be clear to a user. I tried to reword this, to make it clearer but I'm not sure my attempt is any better ...

"Utilising protobuf as a cache encoding provides excellent performance and allows for interoperability between the {brandname} clients available in different programming languages for both the HotRod and REST endpoints."